src/oca_github_bot/webhooks/on_command.py

```diff
@@ -16,5 +19,19 @@
     username = event.data["comment"]["user"]["login"]
     text = event.data["comment"]["body"]
 
-    for command in parse_commands(text):
-        command.delay(org, repo, pr, username)
+    try:
+        for command in parse_commands(text):
+            command.delay(org, repo, pr, username)
+    except CommandError as e:
+        # Add a comment on the current PR, if
+        # the command were misunderstood by the bot
+        add_pr_comment.delay(
+            org,
+            repo,
+            pr,
+            f"Hi @{username}. Your command failed:\n\n"
+            f"``{e}``.\n\n"
+            f"{OCABOT_USAGE}\n\n"
+            f"{OCABOT_EXTRA_DOCUMENTATION}",
+        )
+        raise
```

Comment:
No need to raise here: that will generate unnecessary noise in the logs and sentry. And these logs would not be actionable as it's a user error.

Reply:
done.